op-node/cmd/genesis/cmd.go

```diff
@@ -187,10 +189,23 @@ var Subcommands = cli.Commands{
 				}
 
 				if config.L1StartingBlockTag == nil {
```

Comment:
Do we still want to allow a user to optionally pass in the `L1StartingBlockTag` or should we always retrieve the block num from `SystemConfig.startBlock()`?

Reply:
I don't see a great reason to allow for it given it can be a footgun and I don't see any utility from it

Reply:
Removed that option here: https://github.com/ethereum-optimism/optimism/pull/11181/commits/5a8be9e40d5049a2dd8740a2d0483cbd65a13f4c